pkg/apis/core/validation/validation.go

```diff
@@ -1667,6 +1667,9 @@ func ValidatePersistentVolumeSpec(pvSpec *core.PersistentVolumeSpec, pvName stri
 		allErrs = append(allErrs, field.Required(fldPath.Child("accessModes"), ""))
 	}
 	for _, mode := range pvSpec.AccessModes {
+		if utilfeature.DefaultFeatureGate.Enabled(features.ReadWriteOncePod) && mode == core.ReadWriteOncePod {
+			continue
+		}
 		if !supportedAccessModes.Has(string(mode)) {
```

Comment:
So supportedAccessModes does not include ReadWriteOncePod?

Reply:
No, it just includes `ReadWriteOnce`, `ReadOnlyMany`, and `ReadWriteMany`. I plan on adding `ReadWriteOncePod` to this set once the feature is GA.